src/robotFactory.js

```diff
@@ -1,10 +1,110 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(
+    name,
+    weight,
+    { x = 0, y = 0 },
```

Comment:
```suggestion
    coords = { x = 0, y = 0 },
```
You may declare it here.

Reply:
@qoa11a  It seems like I may not.

![Screenshot 2023-06-27 145636](https://github.com/mate-academy/js_robot_factory/assets/98763083/49c83a48-96cb-44da-9d10-5f97a3b7d6e5)

And if I use a `:` then I won't be able to get default values if an empty object `{}` is passed as an argument.